application/src/main/java/uk/nhs/digital/uec/api/controller/RegionController.java

```diff
@@ -21,16 +21,13 @@
 import uk.nhs.digital.uec.api.model.PostcodeMapping;
 import uk.nhs.digital.uec.api.service.RegionService;
 
-/**
```

Comment:
in future to avoid all these formatting changes, once you have finished your code changes please use vscode to fix the formatting.